streamrouter/__init__.py

```diff
@@ -1,15 +1,27 @@
 from .config import RouterConfig
-from .consul import ConsulClient
+from .consul import (
+    Service,
```

Comment:
Nitpicking: would be nice to order the imports

Btw, didn't we agree not to use the relative imports?

Reply:
Yep, I forgot to order them here. Absolute imports will not work here. They would have to look like this:
```python
from streamrouter.config import RouterConfig
...
```

however, the `streamrouter` module hides the submodules and reexports their contents.